.github/workflows/as400.yml

```diff
@@ -63,11 +63,13 @@ jobs:
 
       - name: Prepare files for packaging
         run: |
+          # Get version from the pom.xml
+          version=$(xmllint --xpath "/*[local-name()='project']/*[local-name()='version']/text()" as400/connector.as400/pom.xml)
```

Comment:
you should add version_file input to get-environment workflow in order to manage et set properly `needs.get-environment.outputs.version`
you need to manage this case in this step : https://github.com/centreon/centreon-plugins/blob/develop/.github/workflows/get-environment.yml#L247
To parse xml version, you can use this kind of parsing : https://github.com/centreon/centreon-modules/blob/develop/.github/workflows/get-environment.yml#L312
or just use a regular regexp
Then, you can remove libxml2 from dockerfiles